payments-core/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -159,17 +162,6 @@ internal class PaymentSheetViewModel @Inject internal constructor(
         }
     }
 
-    fun updatePaymentMethods() {
-        viewModelScope.launch {
-            _paymentMethods.value = customerConfig?.let {
-                paymentMethodsRepository.get(
-                    it,
-                    PaymentMethod.Type.Card
-                )
-            }.orEmpty()
-        }
-    }
-
     fun fetchStripeIntent() {
```

Comment:
now that we are updating payment method within fetchStripeIntent, probably worth while adding some doc to this method